.travis.yml

```diff
@@ -35,14 +35,10 @@ install:
   - cd ..
 
 script:
-  # gcc in the versions that we use has problems with sanitizers
-  - if test "$CXX" = "clang++"; then export CXX_FLAGS="-fsanitize=undefined,address"; fi
-  - cmake -DCMAKE_BUILD_TYPE="$BUILD_TYPE" -DCMAKE_CXX_FLAGS=$CXX_FLAGS .
```

Comment:
So we're now testing only Release build?

Reply:
Sorry for that, I explicitly used Release for testing with CI in my fork.

 At the moment, CI in master does not set build type at all (introduced by commit 5c9b650b3926e01389e19c27164a98ccdc0ab835). I have reverted it so that both Release and Debug builds are tested once again.